src/AppMain.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;700;800&display=swap');
```

Comment:
Instead of bopping this inside the CSS file, I recommend you edit `public/index.html` instead and use a `<link>` tag. You'll get a few benefits in terms of page performance and asynchronous loading there.

(AFAIK Matt is also doing something in #1 to address this?)